cmd/agent/subcommands/launchgui/command.go

```diff
@@ -36,8 +37,7 @@ func Commands(globalParams *command.GlobalParams) []*cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			return fxutil.OneShot(launchGui,
 				fx.Supply(cliParams),
-				fx.Supply(core.BundleParams{
-					ConfigParams: config.NewAgentParams(globalParams.ConfFilePath)}),
+				fx.Supply(command.GetDefaultCoreBundleParams(cliParams.GlobalParams)),
```

Comment:
No extra config ?

Reply:
I updated the `command.GetDefaultCoreBundleParams()` func to use extra config files.
https://github.com/DataDog/datadog-agent/pull/26923/files#diff-4999afb7c68329804af928d035a0d376308440b4185d0273897225cb7ad2490fR56-R60